easybuild/easyconfigs/w/WildMagic/WildMagic-5.17-foss-2018b.eb

```
@@ -0,0 +1,34 @@
+easyblock = "MakeCp"
+
+name = 'WildMagic'
+version = '5.17'
+
+homepage = "https://www.geometrictools.com/index.html"
+description = """Wild Magic 5.17"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'optarch': False}
+
+source_urls = ['https://www.geometrictools.com/Downloads/']
+sources = ['WildMagic5p17.zip']
+checksums = ['b2caf7aa0c5309eb40965ed26439388dc1c93eb632312b40573f83e4746a9ecf']
+
+dependencies = [
+    ('freeglut', '3.0.0'),
+]
+
+build_cmd = "make CFG=ReleaseDynamic -f makefile.wm5"
```

@tstrempel please use buildopts instead:
```suggestion
buildopts = "CFG=ReleaseDynamic -f makefile.wm5"
```